specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md

```diff
@@ -0,0 +1,389 @@
+| GUID_MONITOR_POWER_ON             | The monitor has been powered on or off. From Windows 8 and Windows Server 2012, apps should use GUID_CONSOLE_DISPLAY_STATE instead.                                                                                    | _Not implemented yet_                                                  |
+| GUID_POWER_SAVING_STATUS          | Battery saver has been turned on or off.                                                                                                                                                                               | `EnergySaverStatus`                                                    |
+| GUID_POWERSCHEME_PERSONALITY      | The active power scheme personality has changed. All power schemes map to one of these personalities.                                                                                                                  | `PowerSchemePersonality`                                               |
+| GUID_SESSION_DISPLAY_STATUS       | The display associated with the application's session has been powered on or off.                                                                                                                                      | Covered by GUID_CONSOLE_DISPLAY_STATE.                                 |
+| GUID_SESSION_USER_PRESENCE        | The user status associated with the application's session has changed.                                                                                                                                                 | `UserPresenceStatus`                                                   |
+| GUID_SYSTEM_AWAYMODE              | The system is entering or exiting away mode.                                                                                                                                                                           | `SystemAwayModeStatus` |
+
+The existing PowerManager class exposes only static properties and events: all members are valid to
+be brought over to the Reunion version. Many existing enumerated values are brought over as-is as
+well.
+
+```idl
+namespace Microsoft.Windows.System.Power
+{
+    // Clone to Windows.System.Power.BatteryStatus
+    enum BatteryStatus {
+        NotPresent,
+        Discharging,
+        Idle,
+        Charging
+    }
+
+    // Clone of Windows.System.Power.EnergySaverStatus
+    enum EnergySaverStatus {
+        Disabled,
+        Off,
+        On
+    }
+
+    // Clone of Windows.System.Power.PowerSupplyStatus
+    enum PowerSupplyStatus {
+        NotPresent,
+        Inadequate,
+        Adequate
+    }
+
+    // Status of the device's power source
+    enum PowerSourceKind {
+        AC,
+        DC,
+        ShortTerm
+    }
+
+    // State of the primary display
+    enum DisplayState {
+        Off,
+        On,
+        Dimmed
+    }
+
+    // Indicates whether the system is busy or idle.
+    enum SystemBusyState {
+        Idle,
```

Comment:
DISCUSSION: There's a single pulse, "going from idle to non-idle" which means there's no way to ask the system "are you idle," and there's no way to know that the system is going from non-idle to idle.  This event is the "hey you should go do a bunch of work."  There's no matching property of `SystemBusyState` to query.

Reply:
Fixed.